src/Onboarding/EulaModal.spec.tsx

```diff
@@ -28,31 +31,19 @@ jest.mock("../locales/eula/es_PR.html", () => {
 jest.mock("../locales/eula/ht.html", () => {
   return "ht"
 })
+jest.mock("@react-navigation/native")
 
 describe("EulaModal", () => {
-  it("won't continue until a user accepts the eula", () => {
-    const onPressModalContinueSpy = jest.fn()
-
-    const { getByLabelText } = render(
-      <EulaModal
-        onPressModalContinue={onPressModalContinueSpy}
-        selectedLocale="en"
-      />,
-    )
-    const continueButton = getByLabelText("Continue")
-    fireEvent.press(continueButton)
-
-    expect(onPressModalContinueSpy).not.toHaveBeenCalled()
-    fireEvent.press(getByLabelText("I accept the licensing agreement"))
-    fireEvent.press(continueButton)
-    expect(onPressModalContinueSpy).toHaveBeenCalled()
-  })
-
   it("adds an accessible close button", () => {
+    const navigationSpy = jest.fn()
+    ;(useNavigation as jest.Mock).mockReturnValue({ navigate: navigationSpy })
```

Comment:
Since won't use the spy could this just be:
```suggestion
    ;(useNavigation as jest.Mock).mockReturnValue({ navigate: jest.fn() })
```

Reply:
Good catch!